scripts/_helpers.py

```diff
@@ -10,10 +10,17 @@
 
 import country_converter as coco
 import geopandas as gpd
-import numpy as np
 import pandas as pd
+import yaml
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
+osm_config_path = os.path.join("configs", "osm_config.yaml")
+with open(osm_config_path, "r") as f:
+    osm_config = yaml.safe_load(f)
+world_iso = osm_config["world_iso"]
+continent_regions = osm_config["continent_regions"]
+iso_to_geofk_dict = osm_config["iso_to_geofk_dict"]
+osm_clean_columns = osm_config["osm_clean_columns"]
```

Comment:
What do you think about the possible auxiliary function to handle this?
To me, these code, despite being practical, is a bit ugly and difficult to read unless you know the model.

Having an auxiliary function as described in the past function may be nice maybe.
The function may get a variable number of inputs and returns as many outputs as the inputs + 1.
For example: f("world_iso", "continent_regions") shall return: osm_config, world_iso and continent_regions.

This could be quite handy to load these codes in one go:
osm_config, world_iso, ... = read(...)

In particular, iso_to_geofk and osm_clean_columns may not be constant values here, but dynamically loaded only when necessary, but this is a detail. We can keep them as constants

Reply:
done.